samples/bookinfo/src/mysql/Dockerfile

```diff
@@ -12,7 +12,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM mysql:8.0.3
+FROM mysql:latest
```

Comment:
Using `latest` tag is not a good idea, since it will result in version changes at random points in time without any Istio tracking. I'd suggest explicitly choosing the latest: `8.0.16`. 

Are you sure the latest doesn't still have vulnerabilities? https://hub.docker.com/_/mysql?tab=tags

Reply:
I can change it to the specific latest version for mysql. I debated between using latest vs a specific version. I figured using the 'latest' tag version, we would not have to go and update versions to point to the latest. Regarding security vulnerabilities for the latest mysql version, the image scanner tool shows 3 vulnerabilities still (which is better than the 15 vulnerabilities using the 8.0.3 version that is used in the current version of this Dockerfile). I have mentioned this in the subject of this PR. Hope this helps. Appreciate your feedback and review. Thanks

Reply:
OK thanks, but do change it to `8.0.16`. Using the `latest` tag is never recommended, because it adds unpredictability - you never know when the version Istio is using might change under the covers. Even though a new version is unlikely to break things, you never know, and not testing it before we switch to it is not a good idea.

Also, we don't always need to be using the latest version anyway. Once we find a version that works for our purposes and has no vulnerabilities, we can keep that version and not worry about any newer versions, until we need it.

Reply:
Sounds good, I will make these changes and upload it today itself. Could you take a look at this: https://github.com/istio/istio/blob/master/samples/bookinfo/src/reviews/reviews-wlpcfg/Dockerfile. This is the booking-reviews app Dockerfile. This base tag has 0 vulnerabilities but should I still go ahead and change it to the specific version instead of using the 'latest' tag? Thanks @frankbu

Reply:
Yes please, change that one too. Thanks

Reply:
@frankbu I pushed back on use of the latest tag on the release-1.1 branch because I'm concerned that if we lock down versions we'll forget to update them and become increasingly stale and, ironically, increasingly insecure.

I do see your point though.  I think the ideal is to lock down in the image, but periodically move them forward.  How about I add an item to the next test and release wg meeting agenda to discuss.

I'll bring it to the attention of everyone on this PR

Reply:
It's a good point about not wanting to be stuck with increasingly insecure versions, but I thought that part of the security scan of the github repo recognizes when there is a vulnerability in a library that we're using and makes an annoyingly clear message to that effect, until we upgrade the library.

Reply:
@frankbu who does these security scans of the repo? I have this patch out to use imagescanner tool to run vulnerability scans for images: https://github.com/istio/istio/pull/14363. If there is another way to do this, I would like to take a look at this and see if we can have a consistent way of doing this.